proxy/http/Http1ClientSession.cc

```diff
@@ -547,3 +547,21 @@ Http1ClientSession::attach_server_session(HttpServerSession *ssession, bool tran
     slave_ka_vio = nullptr;
   }
 }
+
+void
+Http1ClientSession::set_session_active()
+{
+  if (!m_active) {
```

Comment:
This doesn't look like a good abstraction to me.
I'd do `virtual void increment_current_active_client_connection_stat() = 0;` or `virtual const char* get_current_active_client_connection_stat_name() = 0;`

Reply:
First suggestion looks good. I'll change with that.